english/marshmallow.md

```diff
@@ -865,5 +865,5 @@ Do you know [Indonesian Culture?](https://en.wikipedia.org/wiki/Culture_of_Indon
 
 Explore the unknown. Most people die without ever visiting this [place.](underwater/scuba-diving.md)
 
-As you attempt this, a voice booms!!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.